FixedPopupPanels/tests/FixedPopupPanels-specs.js

```diff
@@ -1,11 +1,41 @@
 import {FloatingLayerDecorator} from '@enact/ui/FloatingLayer';
 import '@testing-library/jest-dom';
-import {render, screen} from '@testing-library/react';
+import {fireEvent, render, screen} from '@testing-library/react';
+import {useState} from 'react';
```

Comment:
Please move 'react' import before '@testing-library' imports (alphabetically; @ symbol is not taken into consideration).